Pacemaker/clone_experiments/clone_resource.md

```diff
@@ -0,0 +1,346 @@
+     s3server-2:6       (ocf::seagate:s3server):        Started srvnode-2
+     s3server-2:7       (ocf::seagate:s3server):        Started srvnode-2
+     s3server-2:8       (ocf::seagate:s3server):        Started srvnode-2
+     s3server-2:9       (ocf::seagate:s3server):        Started srvnode-2
+     s3server-2:10      (ocf::seagate:s3server):        Started srvnode-2
+ Clone Set: s3server-3-clone [s3server-3] (unique)
+     s3server-3:0       (ocf::seagate:s3server):        Started srvnode-3
+     s3server-3:1       (ocf::seagate:s3server):        Started srvnode-3
+     s3server-3:2       (ocf::seagate:s3server):        Started srvnode-3
+     s3server-3:3       (ocf::seagate:s3server):        Started srvnode-3
+     s3server-3:4       (ocf::seagate:s3server):        Started srvnode-3
+     s3server-3:5       (ocf::seagate:s3server):        Started srvnode-3
+     s3server-3:6       (ocf::seagate:s3server):        Started srvnode-3
+     s3server-3:7       (ocf::seagate:s3server):        Started srvnode-3
+     s3server-3:8       (ocf::seagate:s3server):        Started srvnode-3
+     s3server-3:9       (ocf::seagate:s3server):        Started srvnode-3
+     s3server-3:10      (ocf::seagate:s3server):        Started srvnode-3
+```
+
+### Approach 3: Create clone group.
+  - For s3server Create 11 instance in group and then clone group
+  - It good when scaling cluster.
+  - when clones not move across node and not necessary to start all service in parallel.
+
+```
+    # Move motr_fid.conf to /root/
+    # Assuming all service available
+    # Eg for 3 node and s3sever with 11 instance.
+    Create 11 resource of s3server with group s3server_group and then clone that group.
+
+    $ pcs resource create s3server-1 ocf:seagate:clone-service service=s3server --group s3server
+    $ pcs resource create s3server-2 ocf:seagate:clone-service service=s3server --group s3server
+    $ pcs resource create s3server-3 ocf:seagate:clone-service service=s3server --group s3server
+    $ pcs resource create s3server-4 ocf:seagate:clone-service service=s3server --group s3server
+    $ pcs resource create s3server-5 ocf:seagate:clone-service service=s3server --group s3server
+    $ pcs resource create s3server-6 ocf:seagate:clone-service service=s3server --group s3server
+    $ pcs resource create s3server-7 ocf:seagate:clone-service service=s3server --group s3server
+    $ pcs resource create s3server-8 ocf:seagate:clone-service service=s3server --group s3server
+    $ pcs resource create s3server-9 ocf:seagate:clone-service service=s3server --group s3server
+    $ pcs resource create s3server-10 ocf:seagate:clone-service service=s3server --group s3server
+    $ pcs resource create s3server-11 ocf:seagate:clone-service service=s3server --group s3server
+    $ pcs resource clone s3server
+
+    Clone Set: s3server-clone [s3server]
+      Started: [ srvnode-1 srvnode-2 srvnode-3 ]
+```
+
+# Clone Group
```

Comment:
Is this the recommended approach for LDR_R2? If not, can we please document the recommendations coming out of this POC?

Reply:
Here i only done exploration of resource group and when to use it. currently approach is not finalize.

Reply:
ok